src/pages/auth/login.tsx

```diff
@@ -52,6 +52,9 @@ export default function Login() {
         localStorage.setItem(localStorageKeyForLoginEmail, email);
         router.push(`/auth/verify`);
       }
+    } catch (err) {
+      console.error(`Unkonwn error on sign-in: ${err}`);
+      toast.error(`糟糕，系统错误，请联系管理员：${err}`);
```

Comment:
Can change to:

```
const msg = `糟糕，系统错误，请联系管理员：${err}`;
console.error(msg);
toast.error(msg):
```

So the messages in the console and the toaster are consistent.